### ISSUE TITLE
[WHITE-PAPER] Birdwatch paper and limited explanatory model

### ISSUE THREAD

**Issue by jakorostami** [twitter/communitynotes#192]
Hi.

I just read the Birdwatch white-paper which Community Notes is based on. I realized that in the Appendix, specifically table 10 which relates to RQ2, the model for Tweet agreement on a 5-point scale has limited explanatory power. Modestly speaking. 

![Skärmbild 2024-01-06 145338](https://github.com/twitter/communitynotes/assets/51382407/b5ce164e-2125-4cc7-961e-c32ac5e78b2f)

Using an unweighted OLS with an _**adjusted R-squared**_ of 0.03 but with statistically significant variables (few of them) and a statistically significant F-statistics just means the model is "better" than nothing. In the lens of statistics this model is useless for explainability or to use the estimated dependent variable 'Tweet agreement' in a bivariate association as erroneous.

**The percentage of standard deviation explained is:** 
`1 - sqrt(1 - r_squared)*100 = 2.02%`

In summary, the model can explain about **4% in the variability of Tweet agreement** and about **2% of the standard deviation of its errors**.

Have you thought about redoing the analysis/paper with statistically rigorous methods?

Kindly

**Comment by jakorostami** [comment]
Bump

**Comment by jakorostami** [comment]
Bump.

It's been 1 year since I raised the issue with your Community Notes (former Birdwatch) model.

Anyone care to answer/comment? Perhaps you have changed the model for Community Notes?
